File: PoliGo/api/urls.py
from django.urls import path, include
from api import views


urlpatterns = [
    path('', views.Home),
    path('checkstock/', views.check_stock),
    path('addstock/', views.add_stock),
    path('removestock/', views.remove_stock),
    path('addproduct/', views.add_product),
    path('login/', views.login),
    path('signup/', views.register),
    path('shoppinghistory/', views.shopping_history),
    path('shoppingcart/', views.shopping_cart),
    path('removefromcart/', views.remove_from_cart),
    path('placeorder/', views.place_order),
    path('addcard/', views.add_card),
    path('addtocart/', views.add_to_cart)
]

File: PoliGo-Android/app/src/main/java/com/thethreebees/poligo/URLs.java
package com.thethreebees.poligo;

public class URLs {
    public static final String PROTOCOL = "http";
    public static final String AUTHORITY = "192.168.0.108:8000";
    public static final String ROOT_PATH = "api/v1";

    private static final String ROOT_URL = PROTOCOL + "://" + AUTHORITY + "/" + ROOT_PATH + "/";

    public static final String URL_REGISTER = ROOT_URL + "signup/";
    public static final String URL_LOGIN = ROOT_URL + "login/";
    public static final String URL_ADD_TO_CART = ROOT_URL + "addtocart/";
    public static final String URL_SHOPPING_HISTORY = ROOT_URL + "shoppinghistory/";
    public static final String URL_SHOPPING_CART = ROOT_URL + "shoppingcart/";
    public static final String URL_ADD_CARD = ROOT_URL + "addcard/";


}